roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,35 @@
   with_items: network_interfaces_d
   tags: ['prereboot']
 
+- name: set slrouter address
+  set_fact: sl_router_address="{{ ansible_default_ipv4.network.split('.')[0] }}.{{ ansible_default_ipv4.network.split('.')[1] }}.{{ansible_default_ipv4.network.split('.')[2] }}.{{ (ansible_default_ipv4.network.split('.')[3] | int) + 1 }}"
+  when: ansible_default_ipv4.interface == hostvars[inventory_hostname][primary_interface].device   
+
+- name: add static route for multi subnet
+  add_sys_route: state=present
```

Comment:
So this is to obtain the changed status, which would not be returned by `ip route replace`?

This check if exists and if not create approach creates the potential for a race condition although Ansible seems to depend on it frequently.